CHANGELOG.md

```diff
@@ -3,8 +3,20 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.11.0] - 2023-08-01
 
-## 0.10.0 - 2023-06-23
+### Added
```

Comment:
Let's put the "Added" (features) section below the "Changed" (incompatible) section - https://github.com/nucypher/nucypher-core/blob/8b1ed8c0f541dc7d921b4e87e794a1cfe999d893/PUBLISHING.md?plain=1#L9